source/usage-examples/deleteOne.txt

```diff
@@ -3,3 +3,50 @@ Delete a Document
 =================
 
 .. default-domain:: mongodb
+
+You can delete a single document in a collection using the
+``DeleteOne()`` method.
+
+The following example specifies a query filter that matches documents in
+the ``movies`` collection with the value "Twilight" in the ``title`` field, and
+deletes the first document that matches. 
+
+.. include:: /includes/quick-start/run-example-tip.rst
+
+.. literalinclude:: /includes/quick-start/deleteOne.go
+   :start-after: begin deleteOne
+   :end-before: end deleteOne
+   :emphasize-lines: 2
+   :language: go
+   :dedent:
+
+Click here <TODO> to see a fully runnable example.
+
+Expected Result
+---------------
+
+After deleting this document, you should not be able to find it in the
```

Comment:
```suggestion
After running the above code snippet, you should not be able to find the following document in the
```
Suggestion: I think "after deleting... you shouldn't be able to find" is a tautological statement. I think "running the above code snippet" better conveys the task the just did (which may or may not have actually deleted the document). 

Suggestion: I think "this document" should be "the following" document